### DISCUSSION TITLE
Validation on form preventing use of Chinese characters?

### DISCUSSION THREAD

**Discussion by matthewbcool** [mozilla/hubs#2732] [General Questions & Discussion] [answered]
Will future versions of the client allow for Chinese avatar names?

<img width="477" alt="Screen Shot 2020-07-26 at 1 45 10 PM" src="https://user-images.githubusercontent.com/4493657/88677422-e00e2180-d0b2-11ea-9921-67cace489f3b.png">

**Comment by misslivirose** [comment, marked as answer]
Hi @matthewbcool - we're planning to do a pass on localization work in this quarter. You can follow along with the progress on this issue: https://github.com/mozilla/hubs/issues/2637 - I've linked this question here to ensure that this is a place where we can change how we do username validation for the future as part of this work.
